sdk/core/azure-core/test/libcurl-stress-test/libcurl_stress_test.cpp

```diff
@@ -63,8 +55,9 @@ void Operation(int repetitions)
 int main(int argc, char** argv) // i can have either 0 or 2 params here
 {
   (void)argv; // to get rid of the unused warning
```

Comment:
If you're not going to be using `argv`, then just eliminate it from the parameters:

```cpp
int main(int argc, char**)
```

Reply:
I'm confused. Godbolt says [this compiles](https://www.godbolt.org/z/qe1Tj6aTK):
```c++
int main(int argc, char**) {
    return argc+1;
}
```

Reply:
never seen that declaration before, :) seems hacky but if it works, why not?

Reply:
As far as I know, it's not an uncommon C++ idiom (actually it might even be a C idiom). 

It's how you indicate to the C++ compiler (or more accurately C++ linters) that yes, there's a parameter to this method, but we're not going to be using it).

I use it a lot because it's so useful. And it's WAY less annoying than the "`<variable>;`" trick.